base/prometheus.yaml

```diff
@@ -107,10 +107,7 @@ spec:
         - sidecar
         - --log.level=info
         - --prometheus.url=http://127.0.0.1:9090
-        - --reloader.config-file=/etc/prometheus/prometheus.yaml.tmpl
-        - --reloader.config-envsubst-file=/etc/prometheus-shared/prometheus.yaml
```

Comment:
I think we need to keep those 2  as we templated out `yaml.tmpl` to .yaml  for https://github.com/utilitywarehouse/thanos-manifests/blob/master/example/aws/resources/prometheus.yaml.tmpl#L4

Reply:
Right, I was wondering